packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx

```diff
@@ -0,0 +1,40 @@
+# Punctuation
+
+## Use periods consistently
+
+Keep UX copy scannable by eliminating unnecessary periods.
+Do not use periods on:
+
+- buttons
+- labels
+- tooltips
+- bulleted lists
+
+Use periods on:
+
+- multiple sentences
+- any sentence followed by a link
+- modal body text
+
+![Use periods consistently](/use-periods-consistently-1.jpg)
+
+![Use periods consistently](/use-periods-consistently-2.jpg)
```

Comment:
If we stick with images, I'd just put the example UI in the card and remove the text on the left. Also make sure the TextInput looks like a TextInput, again, removing ambiguity (I wasn't sure what this was at first).